Dockerfile

```diff
@@ -35,6 +35,11 @@ RUN npm run stylelint
 # Upload ixbrlviewer.js to github artifacts
 ARG BUILD_ARTIFACTS_GITHUB_RELEASE_ASSETS=/build/iXBRLViewerPlugin/viewer/dist/ixbrlviewer.js
 
+# Host ixviewer.js on CDN
+RUN mkdir /static_release
+RUN tar -czf /static_release/assets.tar.gz -C /build/iXBRLViewerPlugin/viewer/dist/ .
```

Comment:
Nit: Should you tighten this up, or is it true that we want all of the files from the dir forever and not just the ixviewer.js

Reply:
There is only one file in `/build/iXBRLViewerPlugin/viewer/dist/` and that is the ixviewer.js.